plugins/git-release-manager/src/index.ts

```diff
@@ -34,3 +34,16 @@ export const internals = {
   helpers,
   testHelpers,
 };
+
+export { components, constants, helpers, testHelpers };
```

Comment:
Hmm, we don't really want to export these directly, the idea is that they're being exported via the `internal` export.

I think we might be able to work around it by creating `internals.ts` that re-exports these, and then have the index re-export that as `export * as internals from './internals'`